schema.sql

```diff
@@ -35,4 +35,32 @@ ALTER TABLE animals ADD FOREIGN KEY (species_id) REFERENCES species (id);
 -- Add column species_id which is a foreign key referencing species table --
 
 ALTER TABLE animals ADD COLUMN owner_id INT;
-ALTER TABLE animals ADD FOREIGN KEY (owner_id) REFERENCES owners (id);
\ No newline at end of file
+ALTER TABLE animals ADD FOREIGN KEY (owner_id) REFERENCES owners (id);
+
+--- Create the VETS table ----------
+
+CREATE TABLE vets (
+    id BIGSERIAL NOT NULL PRIMARY KEY,
+    name VARCHAR(100) NOT NULL,
+    age INT NOT NULL,
+    date_of_graduation DATE NOT NULL
+);
+-- create the intermediate table between species and vets---
+
+CREATE TABLE specializations (
+    vets_id INT,
+    species_id INT,
+    PRIMARY KEY (vets_id, species_id),
+    CONSTRAINT fk_vets FOREIGN KEY (vets_id) REFERENCES vets(id),
+    CONSTRAINT fk_species FOREIGN KEY (species_id) REFERENCES species(id)
+);
+
+--create Visits table, intermediate table between animals and vets--
+
+CREATE TABLE visits (
+    vets_id INT NOT NULL,
+    animals_id INT NOT NULL,
```

Comment:
- [ ] Please use a singular naming here. ie `animal_id` instead of `animals_id` It makes more sense because we are storing one animal for every visit.